internal/deepprint/deepprint.go

```diff
@@ -12,16 +12,27 @@
 package deepprint
 
 import (
+	"bytes"
 	"crypto/sha256"
 	"fmt"
 	"io"
 	"reflect"
 )
 
-func Hash(v interface{}) string {
+func Hash(v interface{}) []byte {
 	h := sha256.New()
 	Print(h, v)
-	return fmt.Sprintf("%x", h.Sum(nil))
+	return h.Sum(nil)
```

Comment:
This is to save an allocation. It doesn't seem like the value is actually printed anywhere.